server/auth/types/openid/routes.ts

```diff
@@ -164,6 +166,7 @@ export class OpenIdAuthRoutes {
         const nextUrl: string = cookie.oidc.nextUrl;
         const clientId = this.config.openid?.client_id;
         const clientSecret = this.config.openid?.client_secret;
+        const redirectHash: boolean = cookie.oidc?.redirectHash || false;
```

Comment:
Do we want to define the oidc in the security_cookie similar to saml?

Reply:
When is this set on the cookie/do we need to add code that sets this on the cookie?

Reply:
Updated the definition to be explicit instead of `any`.